### ISSUE TITLE
rebber-plugins: protect chars in custom blocks with titles

### ISSUE THREAD

**Issue by artragis** [zestedesavoir/zmarkdown#349]
# configuration

## remark
```javascript
information: {
      classes: 'custom-block-information',
      title: 'optional',
    },
}
```
## rebber

```
informationCustomBlock: require('rebber-plugins/dist/type/customBlocks'),
```

# markdown

```markdown
[[information| Github, the microsoft code forge]]
| this is github
```

# rendered latex

```tex
\begin{Intormation}[Github, the microsoft code forge]
this is github
\end{Information}
```

# expected
```tex
\begin{Intormation}[Github\protect{,} the microsoft code forge]
this is github
\end{Information}
```

**Comment by pierre-24** [comment]
Yep :)

**Comment by artragis** [comment]
is comma the only char to protect?

**Comment by pierre-24** [comment]
same ass #276 : also square brackets :)

**Comment by artragis** [comment]
according to karnaj we can protect by wrapping the caption between `{{` and `}}`

**Comment by TitiAlone** [comment]
@pierre-24 : Is the resolution done at #279 valid for this case (adding the caption under the block)?

If not, could you confirm `{{` and `}}` would work?

**Comment by pierre-24** [comment]
Because it is not a caption that you put under something (like a table, an image or a code), so the resolution of #279 will not work. But `{{,}}` works, just tested with:

```
\begin{Information}[Github{{,}} the microsoft code forge {{[}}or is it?{{]}}]
this is github
\end{Information}
```

**Comment by TitiAlone** [comment]
@pierre-24 : If I understood well what Karnaj suggested, he thinks that:

```
\begin{Information}[{{Github, the microsoft code forge [or is it?]}}]
this is github
\end{Information}
```

would work. Can you confirm this, because it would be much easier to implement?

**Comment by pierre-24** [comment]
It does work! I feel a bit stupid not to have thought about that ^^

**Comment by TitiAlone** [comment]
I solved the problem quickly, but I'm wondering about:

```
\begin{Information}[{{Github, the microsoft \textbf{code forge} [or is it?]}}]
this is github
\end{Information}
```

Would the command in the title be executed properly (`textbf`)?

**Comment by pierre-24** [comment]
![Screenshot from 2019-08-10 21-19-41](https://user-images.githubusercontent.com/7889675/62826001-c67b7d80-bbb4-11e9-9522-bdfd31b1052c.png)

No problem :)

**Comment by TitiAlone** [comment]
Thanks, the PR is opened and ready to merge!

**Comment by vhf** [comment]
Fixed by #351, thanks!

Let's keep this open until release

**Comment by vhf** [comment]
This has been released as:

- rebber-plugins@2.1.0
- rebber@3.1.5
- zmarkdown@6.1.0
